src/Core/Geometry/MeshPrimitives.cpp

```diff
@@ -11,24 +11,27 @@ namespace Core {
 namespace Geometry {
 
 TriangleMesh makeXNormalQuad( const Vector2& halfExts,
-                              const Utils::optional<Utils::Color>& color ) {
+                              const Utils::optional<Utils::Color>& color,
+                              const bool& generateTexCoord ) {
```

Comment:
not sure if it's efficient to have const bool&, why not just bool ?

Reply:
yes, `bool` is better indeed.

Reply:
https://stackoverflow.com/a/11932667

Reply:
Because this is a const parameter. I said a const value like you. Not a reference.

Reply:
Sorry, I didn't see your message when I wrote mine. I probably forgot to refresh the page when I sent mine. So I wrote a duplicate message.